localization/languages/pt-BR.json

```diff
@@ -35,29 +35,29 @@
         "viewSettings": "Ver configurações",
         "viewReadingList": "Ver listas de leitura",
         "takeScreenshot": "Capturar tela",
-        "clearHistory": "Limpar todo o histórico",
+        "clearHistory": "Limpar histórico",
         "clearHistoryConfirmation": null, //missing translation
```

Comment:
what did you mean by "clearHistoryConfirmation"?

Reply:
The original string is ["Clear all history and browsing data?"](https://github.com/minbrowser/min/blob/master/localization/languages/en-US.json#L39); it shows up in a prompt after you select the option to clear history to confirm that you actually want to do it.

Reply:
<img width="420" src="https://user-images.githubusercontent.com/10314059/73140526-75b2f080-403f-11ea-8cee-6f69fa293096.png">